src/ui-config/marketsConfig.tsx

```diff
@@ -302,6 +304,32 @@ export const marketsData: {
       UI_INCENTIVE_DATA_PROVIDER: AaveV3Goerli.UI_INCENTIVE_DATA_PROVIDER,
     },
   },
+  [CustomMarket.proto_base_v3]: {
+    marketTitle: 'Base',
+    v3: true,
+    chainId: ChainId.base,
+    enabledFeatures: {
+      incentives: true,
+      liquiditySwap: true,
+      collateralRepay: true,
+      debtSwitch: true,
```

Comment:
`liquiditySwap`, `collateralRepay` and `debtSwitch` should be false or just removed here

Reply:
updated in 0acd991084f164af3520d257a3e790ee8fe62230